assets/src/dashboard/app/views/savedTemplates/test/savedTemplates.js

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+/**
+ * External dependencies
+ */
+import { fireEvent } from '@testing-library/react';
+import { SavedTemplatesContent, SavedTemplatesHeader } from '../index';
+import { renderWithTheme } from '../../../../testUtils';
+import { STORY_SORT_OPTIONS, VIEW_STYLE } from '../../../../constants';
+import LayoutProvider from '../../../../components/layout/provider';
+
+const fakeStories = [
+  {
+    id: 1,
+    status: 'publish',
+    title: 'Story A',
+    pages: [{ id: '10' }],
+    centerTargetAction: () => {},
+    bottomTargetAction: () => {},
+  },
+  {
+    id: 2,
+    status: 'draft',
+    title: 'Story B',
+    pages: [{ id: '20' }],
+    centerTargetAction: () => {},
+    bottomTargetAction: () => {},
+  },
+  {
+    id: 3,
+    status: 'publish',
+    title: 'Story C',
+    pages: [{ id: '30' }],
+    centerTargetAction: () => {},
+    bottomTargetAction: () => {},
+  },
+];
+
+jest.mock('../../../../components/previewPage.js', () => () => null);
+jest.mock('../../../../app/font/fontProvider.js', () => ({ children }) =>
+  children
+);
```

Comment:
We mock out the PreviewPage and FontProvider components because they require too many outside dependencies that defeat the purpose of the unit tests.